### ISSUE TITLE
Dimmer 2

### ISSUE THREAD

**Issue by Exerqtor** [fiLLLip/namron-homey#27]
Would be great if you could add support for the "new" Dimmer 2 as well! Should have some improvements over the first one.

If you need one physicaly i cant send you one so you can do your magic, and send it back when your done 😅 can't use the dimmers atm. anyways haha

**Comment by fiLLLip** [comment]
Namron has taken over the development  themselves, so I can't help you here. Post a comment to them on homey app store 👌

**Comment by Exerqtor** [comment]
Yeah i saw the ownership has gotten transfered right after i posted, to bad. Thanks for the work you put in it atleast! 😘
